tool/ci/constants.go

```diff
@@ -0,0 +1,62 @@
+package ci
+
+const (
+	// Assign is the subcommand to assign reviewers
+	Assign = "assign-reviewers"
```

Comment:
I would find it clearer to name these `AssignSubcommand`, `CheckSubcommand`, etc.